browser-switch/build.gradle

```diff
@@ -77,76 +78,73 @@ artifacts {
     archives sourcesJar
 }
 
-signing {
-    required {
-        !version.endsWith("DEVELOPMENT")
-    }
-    sign configurations.archives
-}
+def sonatypeUsername = System.env['SONATYPE_USERNAME']
+def sonatypePassword = System.env['SONATYPE_PASSWORD']
 
 nexusStaging {
     packageGroup = 'com.braintreepayments'
+    stagingProfileId = 'd2424106a9f1f'
```

Comment:
Where did this come from?

Reply:
We had to run `./gradlew getStagingProfile` from the root project. There's supposed to be a way where the plugins get the stagingProfileId dynamically, but setting it statically here helped us get it to work.

We should be able to refine the gradle file a bit more in the future.

Reply:
> The Profile ID displays the unique identifier that can be used for staging to this repository using the Nexus Staging Maven plugin. 

>  Explicit or Implicit is the default behavior and causes the repository manager to select the profile by the provided staging profile identifier and to fall back to an automatic determination, if none is provided. It is necessary to be used with the Maven deploy plugin and the correct staging profile is determined using repository targets together with the generic deploy URL.

([docs reference](https://help.sonatype.com/repomanager2/staging-releases/configuring-the-staging-suite#ConfiguringtheStagingSuite-Overview))

In short, we weren't able to successfully deploy without it. Though the reasoning for requiring it is still fuzzy to me.